doc/api/events.md

```diff
@@ -344,16 +344,23 @@ added: v3.2.0
 
 Returns the number of listeners listening to the event named `eventName`.
 
-### emitter.listeners(eventName)
+### emitter.listeners(eventName[, unwrap])
 <!-- YAML
 added: v0.1.26
 changes:
   - version: v7.0.0
     pr-url: https://github.com/nodejs/node/pull/6881
     description: For listeners attached using `.once()` this returns the
                  original listeners instead of wrapper functions now.
+  - version: REPLACEME
+    pr-url: REPLACEME
+    description: Second optional argument `unwrap` allows the wrapped
+                 listeners to be returned (such as ones created with
+                 `.once()`)
 -->
-- `eventName` {any}
+* `eventName` {any} The name of the event.
+* `unwrap` {boolean} When set to true, will return the original listeners
```

Comment:
Not generally a fan of Boolean args in public APIs. Perhaps having a separate `unwrappedListeners()` method would work?

Reply:
I think it would need to be `wrappedListeners` or something similar. Will think on it.

Reply:
If you want an option argument, I'd prefer to be explicit and call it `unwrapOnceListeners` (which could also be a string or object instead of a boolean).

However, from a performance point of view new methods for separate input types should always be a good idea.

Reply:
This should be addressed now, instead we have a `wrappedListeners` method on EventEmitter.